homeassistant/components/homematicip_cloud/strings.json

```diff
@@ -23,7 +23,7 @@
     "abort": {
       "unknown": "Unknown error occurred.",
       "connection_aborted": "Could not connect to HMIP server",
```

Comment:
And possibly this.
```suggestion
      "connection_aborted": "[%key:common::config_flow::error::cannot_connect%]",
```

Reply:
I didn't change that one because the parent of this 'connection_aborted' is 'abort' and you are refering to a 'cannot_connect' key from 'error'? Not sure if that is okay.

Reply:
that's fine.